.isort.cfg

```diff
@@ -1,15 +1,24 @@
 [settings]
 multi_line_output=3
-include_trailing_comma=True
+include_trailing_comma=true
 force_grid_wrap=0
 use_parentheses=True
 line_length=88
-ensure_newline_before_comments=True
-known_third_party=omegaconf,ray,pytest,typing_extensions
-known_first_party=hydra
+ensure_newline_before_comments=true
+known_libs=pkg_resources,setuptools
+sections=
+    FUTURE
+    STDLIB
+    LIBS
+    THIRDPARTY
+    FIRSTPARTY
+    LOCALFOLDER
+default_section=THIRDPARTY
+known_first_party=hydra,hydra_app,configen,example
 skip=
     plugins
+    ,__init__.py
```

Comment:
Why do you need to skip `__init__.py`?

Reply:
No problem! isort doesn't default skip `__init__.py` anymore: https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0/#not_skip

Reply:
I see. as a followup, can you experiment with not excluding __init__.py from isort and see if we can make that work?

Reply:
Sure.

Reply:
Removed skip of **init**.py from the config and skip list and tests pass.